_format/1.1/index.md

```diff
@@ -164,6 +169,8 @@ In addition, a resource object **MAY** contain any of these top-level members:
 * `links`: a [links object][links] containing links related to the resource.
 * `meta`: a [meta object][meta] containing non-standard meta-information about a
   resource that can not be represented as an attribute or relationship.
+* `lid`: a local identifier that, together with `type`, uniquely
+  identifies a resource within the document.
```

Comment:
Should it be in the scope of the spec to note the value of UUIDs in solving this problem? More specifically, generating a UUID locally and posting that to a server allows you to ensure that the id on the server is the same value without any real chance of conflict.

Reply:
I agree that this would make a good non-normative note 👍  I think we may mention client-generated IDs in a couple places, but mentioning it near `lid`s makes sense as well.

Reply:
Thanks for asking this, @EndangeredMassa, it made me ask a question at #1244:  https://github.com/json-api/json-api/pull/1244#issuecomment-365557627

Reply:
IMO this should just be called `local`, jsonapi doesn't truncate words anywhere else.

Reply:
@krainboltgreene `id` is truncated from `identifier`

i could see using `localId` over `lid` though ...

Reply:
@les2 Uh, let’s avoid using camel “localId” or snake case “local_id” in the spec. People will go crazy about these things. 😉 lid is fine, IMO.